SGPlayer/Classes/Catalyst.h

```diff
@@ -0,0 +1,159 @@
+//
```

Comment:
In [my fork](https://github.com/MaddTheSane/SGPlayer/tree/madds), I migrated to `simd_float4x4` found in `simd/simd.h`. Granted, I had to re-write [some](https://github.com/MaddTheSane/SGPlayer/blob/e8332fb03782ad9c67c9bb684cfb063efe787134/SGPlayer/Classes/Core/SGVR/SGVector.h) of the functions used, but it should be able to use vector instructions on any architecture supported by LLVM.